code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -81,8 +79,8 @@ datum/reagent/synthflesh
 datum/reagent/synthflesh/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
 	if(!M) M = holder.my_atom
 	if(method == TOUCH)
-		M.adjustBruteLoss(-(1*volume)*REM)
-		M.adjustFireLoss(-(1*volume)*REM)
+		M.adjustBruteLoss(-0.5*volume*REM)
+		M.adjustFireLoss(-0.5*volume*REM)
 		M << "<span class='notice'>You feel your burns healing and your flesh knitting together!</span>"
 	..()
```

Comment:
Pretty pointless to use at 1 per unit; 0.5 per unit will be even worse. Should be 1.5 per unit---especially considering this has no per cycle benefit.